code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,254 @@
+/*
+/datum/bounty/item/assistant/strange_object
+	name = "Strange Object"
+	description = "[current_map.company_name] has taken an interest in strange objects. Find one in maint, and ship it off to [current_map.boss_short] right away."
+	reward = 1200
+	wanted_types = list(/obj/item/relic)
+
+/datum/bounty/item/assistant/scooter
+	name = "Scooter"
+	description = "[current_map.company_name] has determined walking to be wasteful. Ship a scooter to [current_map.boss_short] to speed operations up."
+	reward = 1080 // the mat hoffman
+	wanted_types = list(/obj/vehicle/ridden/scooter)
+	include_subtypes = FALSE
+
+/datum/bounty/item/assistant/skateboard
+	name = "Skateboard"
+	description = "[current_map.company_name] has determined walking to be a wasteful. Ship a skateboard to [current_map.boss_short] to speed operations up."
+	reward = 900 // the tony hawk
+	wanted_types = list(/obj/vehicle/ridden/scooter/skateboard)
+*/
+/datum/bounty/item/assistant/stunprod/New()
+	..()
+	name = "Stunprod"
+	description = "[current_map.boss_short] demands a stunprod to use against dissidents. Craft one, then ship it."
```

Comment:
Not sure if this is the best description to use. Perhaps a more subtle description is needed.

Reply:
Done